### DISCUSSION TITLE
adaptive_lighting.apply takes 10 secs

### DISCUSSION THREAD

**Discussion by liksvette** [basnijholt/adaptive-lighting#393] [Q&A]
I'm trying to use a motion sensor to trigger AL to quickly transition the lights to the current AL setting by using adaptive_lighting.apply, but nothing I do in the config seems to change the time it takes to transition the lights. Always about 10 seconds. 
Any idea what which setting is supposed to help and what I could do to change transition times?

**Comment by th3w1zard1** [comment]
Post your adaptive-lighting config and set logging to debug if you want help debugging this further.
We also need to know what hass version you're using, the adaptive-lighting version, and what type of light it is. If it's a zigbee light, you need to post your `devices.yaml`, `configuration.yaml`, and `groups.yaml` from Z2M.
